Article/Article.js

```diff
@@ -22,8 +55,17 @@ class Article {
 
 - Select all classes named ".article" and assign that value to the articles variable.  
 
+
 - With your selection in place, now chain .forEach() on to the articles variable to iterate over the articles NodeList and create a new instance of Article by passing in each article as a parameter to the Article class.
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll(".article");
+
+articles.forEach((articleElement) => {
+  return new Article(articleElement);
+});
```

Comment:
Small thing here, but if you wanted to speed up the selector speed, you could do something like this:
```js
const articles = document.getElementsByClassName('article');
for (const article of articles) {
  new Article(article);
}
```